server/chat-plugins/repeats.ts

```diff
@@ -229,7 +231,7 @@ export const commands: Chat.ChatCommands = {
 
 		Repeats.addRepeat(room, {
 			id: topic as ID,
-			phrase: roomFaqs[room.roomid][topic],
+			phrase: roomFaqs[room.roomid][topic]?.source,
```

Comment:
We should validate that the rfaq exists and display an error message to the user instead of optional chaining here.